### PR TITLE
feat: persisting open files on refresh/reload

### PR DESCRIPTION
**Pull request by ChoukseyKhushbu**
Fixes issue #7 
- added collections in IndexedDB for last opened files and active files(editor wise)
- queried the collections on `loadFile` action in `Files` module.
- modified `Editor` module `actions` accordingly.

Please review @haxzie.

### CODE REVIEW

**Comment by haxzie** [comment]
Hey @ChoukseyKhushbu, good job with the PR :+1: 

A couple of changes that I think might be good.
1. Rename `openFileFooter.model.js` to `openFileFootprint.model.js`
2. While closing a file, check if the file is active, or check if it exists in the `db.activeFiles` and emove the file from `db.activeFiles`.

Rest everything looks good to me :+1:

File: src/models/openFileFooter.model.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default class OpenFileFootprint {
```
Comment (haxzie) [inline src/models/openFileFooter.model.js:1]: It'd be great if the filename matched the classname

**Review by haxzie — CHANGES_REQUESTED** [review]

**Comment by ChoukseyKhushbu** [comment]
I have implemented the requested changes. Please review @haxzie. Thanks :)

**Review by haxzie — APPROVED** [review]

**Comment by haxzie** [comment]
Good job @ChoukseyKhushbu :+1:  Thanks for the PR :partying_face:

**Pull request closed without merging**